jetty-server/src/main/java/org/eclipse/jetty/server/handler/gzip/GzipHandler.java

```diff
@@ -400,6 +402,24 @@ protected HttpField getVaryField()
 
     /* ------------------------------------------------------------ */
     /**
+     * @return size in bytes of the buffer to inflate compressed request, or 0 for no inflation.
```

Comment:
Should request inflation have its own Handler?   maybe but I think this is not bad either.